torch/_inductor/codecache.py

```diff
@@ -923,6 +927,12 @@ def iterate_over_candidates() -> Generator[CompiledFxGraph, None, None]:
 
         GraphLowering.save_output_code(code)
         output_code_log.debug("Output code: \n%s", code)
+        # On cache hit, use artifact path as filename
+        trace_structured(
+            "inductor_output_code",
+            lambda: {"filename": artifact_path},
```

Comment:
should we add something to indicate that this was a result of cache hit?

Reply:
I don't want to change the inductor_output_code interface just for FXGraphCache: I think it belongs in CompilationMetrics instead. We can work on some cache stats in compilation metrics to include both AOTAutogradCache and FXGraphCache, and log that directly via tlparse's compilation metrics parsing.